dna_c_binding/src/lib.rs

```diff
@@ -154,6 +154,199 @@ _xa_str!(
     holochain_dna_set_dna_spec_version
 );
 
+#[repr(C)]
+pub struct CStringVec {
+    len: usize,
+    ptr: *const *const c_char,
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn holochain_dna_get_zome_names(ptr: *mut Dna, string_vec: *mut CStringVec) {
+    match catch_unwind(|| {
```

Comment:
Please don't. Using `catch_unwind` while possible is totally discuaged. You might as well just early return instead of the assert

Reply:
Hm, I've cargo-culted that over from @neonphog's code..